dev/NavigationView/NavigationView.h

```diff
@@ -75,22 +75,81 @@ class NavigationView :
 
     void CoerceToGreaterThanZero(double& value);
 
-    void OnNavigationViewItemInvoked(const winrt::NavigationViewItem& nvi);
```

Comment:
Can you take a quick look and see if all these need to be public ? Most of them look like implementation details and can be private.

Reply:
Most of them need to be public (except for the repeater on focus callback, which i moved)